gamemode/modules/main_menu/cl_mainmenu.lua

```diff
@@ -0,0 +1,192 @@
+RVR.MainMenu = RVR.MainMenu or {}
+local mainMenu = RVR.MainMenu
+
+mainMenu.darkerYellow = Color( 172, 138, 90 )
+mainMenu.darkBrown = Color( 63, 43, 0 )
+
+local camPos = Vector( 0, 0, 300 )
+local camAng = Angle( 12, 135, 0 )
+
+mainMenu.MENU_START = 0
+mainMenu.MENU_CREATE = 1
+mainMenu.MENU_JOIN = 2
+mainMenu.MENU_MODEL = 3
+
+mainMenu.backgrounds = mainMenu.backgrounds or {}
+
+surface.CreateFont( "RVR_StartMenuButton", {
+    font = "Bungee Regular",
+    size = ScrH() * 0.09,
+    weight = 700,
+} )
+
+surface.CreateFont( "RVR_StartMenuLabel", {
+    font = "Bungee Regular",
+    size = ScrH() * 0.07,
+    weight = 700,
+} )
+
+surface.CreateFont( "RVR_StartMenuSmall", {
+    font = "Bungee Regular",
+    size = ScrH() * 0.05,
+    weight = 700,
+} )
+
+surface.CreateFont( "RVR_StartMenuTextEntry", {
+    font = "Roboto",
+    size = ScrH() * 0.04,
+    weight = 700,
+} )
```

Comment:
Might be nice to pull these out into a `fonts.lua`

Reply:
Will do in the future, project wide. Made a card